app/assets/javascripts/components/description/DescriptionContent.js

```diff
@@ -24,12 +25,15 @@ const replaceStoryLink = (domNode, linkedStories) => {
 }
 
 const DescriptionContent = ({ description, isReadonly, linkedStories, onClick }) => {
+  const converter = new showdown.Converter();
+  const descriptionHTML = converter.makeHtml(description);
+
   const isEmpty = (!description || !description.length);
-  description = Parser(description, { replace: domNode =>
+  description = Parser(descriptionHTML, { replace: domNode =>
     replaceStoryLink(domNode, linkedStories)
   });
   return(
-    (isEmpty) ? editButton(isReadonly) : <div className='description'>{ description }</div>
+    (isEmpty) ? editButton(isReadonly) : <div className='description'> { description }</div>
```

Comment:
There is a space before `{ description }`, why did you put it?

Reply:
Sorry, it was my mistake. Corrected